packages/components/src/Components/SkeletonTable/SkeletonTable.js

```diff
@@ -47,12 +56,16 @@ SkeletonTable.propTypes = {
     sortBy: PropTypes.shape({
         index: PropTypes.number,
         direction: PropTypes.oneOf([ 'asc', 'desc' ])
-    })
+    }),
+    onSelect: PropTypes.bool,
+    canSelectAll: PropTypes.bool
 };
 
 SkeletonTable.defaultProps = {
     rowSize: 0,
-    paddingColumnSize: 0
+    paddingColumnSize: 0,
+    canSelectAll: false,
+    onSelect: false
```

Comment:
Empty function instead of bool
```suggestion
    onSelect: () => undefined
```